frontend/packages/kubevirt-plugin/integration-tests/tests/models/kubevirtDetailView.ts

```diff
@@ -50,8 +53,12 @@ export class KubevirtDetailView extends DetailView {
 
   async removeDisk(name: string) {
     await this.navigateToTab(TABS.DISKS);
+    const count = await resourceRows.count();
     await kubevirtDetailView.selectKebabOption(name, 'Delete');
     await confirmAction();
+    do {
+      await browser.sleep(50);
+    } while ((await resourceRows.count()) !== count - 1);
```

Comment:
is there some timeout which would trigger failure if this action fails to remove the disk?

Reply:
Yes, the timeout of the spec that executes this, but I have changed it to adress Marek's review, so this is no longer relevant